<?xml version="1.0" encoding="UTF-8"?><testrun name="MainRunnableTest" project="Hilos24" tests="1" started="1" failures="1" errors="0" ignored="0">
  <testsuite name="caja2024clientesColaMArketfuture07.MainRunnableTest" time="0.061" displayname="MainRunnableTest" uniqueid="[engine:junit-jupiter]/[class:caja2024clientesColaMArketfuture07.MainRunnableTest]">
    <testcase name="test" classname="caja2024clientesColaMArketfuture07.MainRunnableTest" time="0.061" displayname="test()" uniqueid="[engine:junit-jupiter]/[class:caja2024clientesColaMArketfuture07.MainRunnableTest]/[method:test()]">
      <failure>
        <expected>3</expected>
        <actual>2</actual>
        org.opentest4j.AssertionFailedError: expected: &lt;3&gt; but was: &lt;2&gt;&#13;
	at org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:151)&#13;
	at org.junit.jupiter.api.AssertionFailureBuilder.buildAndThrow(AssertionFailureBuilder.java:132)&#13;
	at org.junit.jupiter.api.AssertEquals.failNotEqual(AssertEquals.java:197)&#13;
	at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:150)&#13;
	at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:145)&#13;
	at org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:527)&#13;
	at caja2024clientesColaMArketfuture07.MainRunnableTest.test(MainRunnableTest.java:20)&#13;
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)&#13;
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)&#13;
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)&#13;
&#13;

      </failure>
    </testcase>
  </testsuite>
</testrun>
